### PR TITLE
Update important-upgrade-notes.rst

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/administration/important-upgrade-notes.rst [hunk 1]
```diff
@@ -7,6 +7,11 @@ Important Upgrade Notes
...
 +====================================================+==================================================================================================================================================================+
+| v5.14.0                                            | **Enable Plugin Health Check** removed from ``config.json``. Instead, all plugins are now automatically monitored and restarted if a plugin has crashed or is    |
+|                                                    | deemed unhealthy. If a plugin crashes three times within an hour, the server automatically deactivates the plugin.                                               |
+|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                                    | Change related to viewing webhooks                                                                                                                               |
```
Comment (amyblais) [inline source/administration/important-upgrade-notes.rst]: Needs help with a description

**Comment by amyblais** [comment]
@wiersgallak Yes, I need help for the one you mentioned here: https://community-daily.mattermost.com/core/pl/um77cnqy4pnczghc1ko6f85e9c. It needs more details.

**Comment by lieut-data** [comment]
@amyblais / @wiersgallak: the PR in question is effectively a security improvement, and could perhaps be summarized as:
> Webhooks are now only displayed if the user is the creator or a system administrator.

**Comment by wiersgallak** [comment]
Thanks @lieut-data I didn't have the opportunity to get back to this yesterday.

**Pull request merged**